include/umf/providers/provider_level_zero.h

```diff
@@ -17,20 +17,56 @@ extern "C" {
 typedef struct _ze_device_handle_t *ze_device_handle_t;
 typedef struct _ze_context_handle_t *ze_context_handle_t;
 
-/// @brief Level Zero Memory Provider settings struct
-typedef struct level_zero_memory_provider_params_t {
-    ze_context_handle_t
-        level_zero_context_handle; ///< Handle to the Level Zero context
-    ze_device_handle_t
-        level_zero_device_handle; ///< Handle to the Level Zero device
-
-    umf_usm_memory_type_t memory_type; ///< Allocation memory type
-
-    ze_device_handle_t *
-        resident_device_handles; ///< Array of devices for which the memory should be made resident
-    uint32_t
-        resident_device_count; ///< Number of devices for which the memory should be made resident
-} level_zero_memory_provider_params_t;
+struct umf_level_zero_memory_provider_params_t;
+
+/// @brief handle to the parameters of the level zero memory provider.
+typedef struct umf_level_zero_memory_provider_params_t
+    *umf_level_zero_memory_provider_params_handle_t;
+
+/// @brief  Create a struct to store parameters of the level zero memory provider.
```

Comment:
please use capital letters in comments referring to UMF Level Zero Memory Provider, as it is a proper noun (we should probably cleanup this in other places in our code / documentation in separate PRs)

Reply:
Done